Dev

desht opened the pull request:
A minor GUI fix and translation tweak
Added a method to help with upcoming FTB Ultimine sidebar support
Updated deps

desht committed:
fix: fix little GUI "holes" in GuiHelper#drawBorderedPanel()

desht committed:
chore: code cleanup in ClientUtils

desht committed:
chore: added EditConfigScreen#setOpenPrevScreenOnClose

desht committed:
build: version -> 2101.1.1

desht committed:
chore: adjust a bit of translation text

desht committed:
build: updated dep versions to latest

MichaelHillcox commented on common/src/main/java/dev/ftb/mods/ftblibrary/ui/GuiHelper.java:
```diff
@@ -51,6 +54,9 @@ public void scissor(Window screen) {
 		}
 	}
 
+	private static final BiFunction<Color4I,Boolean,Color4I> BRIGHTEN = Util.memoize((col,outset) -> col.addBrightness(outset ? 0.15f : -0.1f));
```
Spaces?

```diff
-private static final BiFunction<Color4I,Boolean,Color4I>
+private static final BiFunction<Color4I, Boolean, Color4I>
```

Same with the memo args :P

MichaelHillcox reviewed (approved):
Only one formatting note :+1:

Commit:
chore: big codebase cleanup

Reformatting pass (IntelliJ Code -> Reformat Code)
-  Mostly Tab -> Space conversion
Removed almost all instance of `static { }` initializer blocks
  in favour of `Util.make()` where needed
